.github/workflows/conventional-commit-checker.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - main
+      - kirkstone
+    types:
+      - opened
+      - edited
+      - synchronize
+jobs:
+  lint-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # check for the most recent release: https://github.com/CondeNast/conventional-pull-request-action/releases
+      # replace vX.X.X below with the most recently released version
+      - uses: CondeNast/conventional-pull-request-action@v0.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # to override config-conventional rules, specify a relative path to your rules module, actions/checkout is required for this setting!
+          commitlintRulesPath: "./commitlint.rules.js" # default: undefined
```

Comment:
If I understand the documentation of the action correctly, we need to provide this file in the repo then as well, or keep it undefined for default behavior, don't we?

Reply:
only if we intend to override some rules (in future). in my local test setup I didn't need that file.